pages/guestpage.js

```diff
@@ -14,13 +13,8 @@ export default function GuestPage() {
           <BigFontStyle className="BigFontStyle">
             Freunde & Familie
           </BigFontStyle>
-          <Line>________________________________________</Line>
-          <Button type="button" className="MediumFontStyle" value="add guest">
-            + Gast hinzufügen
-          </Button>
-          <p>
-            <Image src={plusIcon} alt="Plus" width={40} height={40} />
-          </p>
+          <Line></Line>
```

Comment:
You should think about using a `hr` tag for a horizontal line, [see here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr)

Reply:
thanks for the advice!